quizup/src/Routes/BrowseMusic/BrowseMusic.css

```diff
@@ -0,0 +1,38 @@
+.BrowseMusic-section {
+  display: flex;
+  column-gap: 25rem;
+  align-items: center;
+  justify-content: center;
+  margin-top: 15rem;
+  margin-bottom: 10rem;
+}
+.musicOne-quiz,
+.musicTwo-quiz {
+  width: 20%;
+  cursor: pointer;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px,
+    rgba(0, 0, 0, 0.3) 0px 30px 60px -30px,
+    rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
```

Comment:
here as well, try to use variables for colors, and try to use rem instead of px in situations of more than 10px for better responsiveness